interface/ceed-basis.c

```diff
@@ -639,8 +643,13 @@ int CeedSymmetricSchurDecomposition(Ceed ceed, CeedScalar *mat,
     // norm of v[i:m] after modification above and scaling below
     //   norm = sqrt(v[i]*v[i] + sigma) / v[i];
     //   tau = 2 / (norm*norm)
-    tau[i] = 2 * v[i]*v[i] / (v[i]*v[i] + sigma);
-    for (CeedInt j=i+1; j<n-1; j++) v[j] /= v[i];
+    if (fabs(sigma) > 1e-16) {
+      tau[i] = 2 * v[i]*v[i] / (v[i]*v[i] + sigma);
+      for (CeedInt j=i+1; j<n-1; j++)
+        v[j] /= v[i];
+    } else {
+      tau[i] = 0;
+    }
```

Comment:
@jedbrown, with denationalization of mass and stiffness matrices we get division my zero sometimes here. I'm switching to the Householder vector computation in Golub and van Loan which checks for that division by zero, but I'm probably missing something small because a few matrix entries are off by a sign.